int32[] coordinate_mission
int32 cup_color_mission # 0 for green 1 for red
---
int32[] coordinate_camera
int32 cup_color_camera # 0 for green 1 for red